app/controllers/organizations_controller.rb

```diff
@@ -52,6 +39,13 @@ def generate_new_secret
 
   private
 
+  def handle_name_too_long
+    log_image_data_to_datadog
+  rescue Errno::ENAMETOOLONG
```

Comment:
Is this needed? you are calling this method on rescue of this error already

Reply:
`log_image_data_to_datadog` method raises an `Errno::ENAMETOOLONG` exception